cluster/gce/ubuntu/node.yaml

```diff
@@ -0,0 +1,281 @@
+From nobody Tue Aug 11 10:13:54 2015
+Content-Type: multipart/mixed; boundary="===============6024533374511606659=="
+MIME-Version: 1.0
+
+--===============6024533374511606659==
```

Comment:
It is not required by ubuntu, but a quite useful tool. The major benefit of MIME is packing different formats and multiple configuration files into one file. This is important in GCE. We pass the configuration file as "user-data", which does not support multiple files.